UnityProject/Assets/Scripts/UI/PlayerHealthUI.cs

```diff
@@ -52,27 +66,74 @@ void UpdateMe()
 				}
 				return;
 			}
-
-			if (PlayerManager.LocalPlayerScript.playerHealth.IsRespiratoryArrest && !oxygenAlert.gameObject.activeInHierarchy)
+			hasOxygen = !PlayerManager.LocalPlayerScript.playerHealth.IsRespiratoryArrest && !PlayerManager.LocalPlayerScript.playerHealth.respiratorySystem.IsSuffocating;
+			if (!hasOxygen && !oxygenAlert.gameObject.activeInHierarchy)
 			{
 				oxygenAlert.gameObject.SetActive(true);
 			}
 
-			if (!PlayerManager.LocalPlayerScript.playerHealth.IsRespiratoryArrest && oxygenAlert.gameObject.activeInHierarchy)
+			if (hasOxygen && oxygenAlert.gameObject.activeInHierarchy)
 			{
 				oxygenAlert.gameObject.SetActive(false);
 			}
 		}
 
+		if (monitorTemp && PlayerManager.LocalPlayer != null)
+		{
+			if (PlayerManager.LocalPlayerScript.IsGhost || PlayerManager.LocalPlayerScript.playerHealth.IsDead)
+			{
+				if (tempAlert.gameObject.activeInHierarchy)
+				{
+					tempAlert.gameObject.SetActive(false);
+				}
+				return;
+			}
+			tempToggle = PlayerManager.LocalPlayerScript.playerHealth.isBurned;
+			if (tempToggle && !tempAlert.gameObject.activeInHierarchy)
+			{
+				tempAlert.gameObject.SetActive(true);
+			}
+			if (!tempToggle && tempAlert.gameObject.activeInHierarchy)
+			{
+				tempAlert.gameObject.SetActive(false);
+			}
+
+		}
+
+		if (monitorPressure && PlayerManager.LocalPlayer != null)
+		{
+			if (PlayerManager.LocalPlayerScript.IsGhost || PlayerManager.LocalPlayerScript.playerHealth.IsDead)
+			{
+				if (pressureAlert.gameObject.activeInHierarchy)
+				{
+					pressureAlert.gameObject.SetActive(false);
+				}
+				return;
+			}
+			pressureToggle = PlayerManager.LocalPlayerScript.playerHealth.respiratorySystem.pressureStatus;
+			if (pressureToggle != 0 && !pressureAlert.gameObject.activeInHierarchy)
+			{
+				pressureAlert.gameObject.SetActive(true);
+			}
+			if (pressureToggle == 0 && pressureAlert.gameObject.activeInHierarchy)
+			{
+				pressureAlert.gameObject.SetActive(false);
+			}
+			
+
+		}
+
 		if (PlayerManager.LocalPlayer != null)
 		{
 			if (PlayerManager.Equipment.HasInternalsEquipped() && !oxygenButton.IsInteractable())
 			{
+				Logger.Log("Has Gear Equipped");
```

Comment:
Use categories for all your logs so these can be turned off on demand
```suggestion
				Logger.Log("Has Gear Equipped", Category.UI);
```

Reply:
These were for debug only. I have removed them